File: js/stockPicker.js
exports.picker = function(prices) {
    let earningPotential = 0
    let finalArr = []
    let buyDay = []
    let sellDay = []
    for(let i = 0; i < prices.length; i++){
        for(let j = i; j < prices.length; j++){
            if(prices[i] - prices[j] < earningPotential){
                earningPotential = prices[i] - prices[j]
                buyDay.push(i)
                sellDay.push(j)
            }
        }
    }
    finalArr.push(buyDay[0])
    finalArr.push(sellDay[sellDay.length - 1])
    return finalArr
}

File: python/stock_picker.py
def picker(prices):
    earning_potential = 0
    final_arr = []
    buy_day = []
    sell_day = []

    for i in range(0,len(prices),1):
        for j in range(i, len(prices), 1):
            if prices[i] - prices[j] < earning_potential:
                earning_potential = prices[i] - prices[j]
                buy_day.append(i)
                sell_day.append(j)
    final_arr.append(buy_day[0])
    final_arr.append(sell_day[-1])
    return final_arr
